apb.yml

```diff
@@ -17,6 +17,10 @@ metadata:
     - "docker.io/centos/postgresql-96-centos7:9.6"
     - "docker.io/aerogear/ups-config-operator:master"
     - "docker.io/openshift/oauth-proxy:v1.1.0"
+  sdk-docs-android: "http://docs.aerogear.org"
```

Comment:
are these the only urls we have? The url doesn't seem to work. Are there more specific urls that could be used?